### DISCUSSION TITLE
北美矿业挖fil币到底靠不靠谱

### DISCUSSION THREAD

**Discussion by hredniuniu** [filecoin-project/community-china#102] [新手上路]
北美矿业拥有遍布全球19个大型矿场，可同时运行80万台矿机，是全球算力第一矿池，手机一站式云算力挖矿,专注BTC'ETH'FIL等主流货币挖矿服务团队为你解决各种疑惑和问题。
矿机租赁时长6年（2190天）
产出效益高出同行30%-50%
每天收益,随时提现
满算力,无需质押币
专注云算力挖矿,无需购买矿机就能轻松挖矿。
北美矿业官网：www.napmine.com

北美矿业NAP挖矿app下载链接 ： http://nap.sleon.cn/index.html#/register?inviteCode=ehymoc

北美矿业2019年成立于美国波士顿, 已获得美国、澳大利亚等多国合规牌照,实体办事处位于美国马萨诸州波士顿肯德尔广场, 绝无跑路风险。其母公司⸺北美智科国际集团（NAZK),由美国、加拿大顶级投资机构⸺CIBC、Scotia Bank、National Bank、黑石集团联合创办，主要投产于生命科技、人工智能、航天探测和区块链领域，北美智科国际集团是经美国商务部认证的高新技术企业, 旗下多条产品线获得美国FDA检验认证、欧盟CE认证, 安全可靠、实力过硬！

**Comment by hredniuniu** [comment]
推荐码ehymoc

**Reply by hredniuniu** [reply]
联系方式微信：759042854

**Reply by KaixuanZhao** [reply]
RRMine，雅典娜矿池，全网算力领先1000PB，需要B端C端挖矿方案的滴滴我，欢迎加入社群Wechet：kkmmjjnn00

**Comment by beck-8** [comment]
标题党，指定不靠谱

**Reply by hredniuniu** [reply]
靠不靠譜先進群看看

**Reply by hredniuniu** [reply]
如果要是存在不靠譜歡迎指點

**Reply by KaixuanZhao** [reply]
RRMine，雅典娜矿池，全网算力领先1000PB，需要B端C端挖矿方案的滴滴我，欢迎加入社群Wechet：kkmmjjnn00

**Comment by bigbeer1** [comment]
北美不是已经跑路了？

**Reply by KaixuanZhao** [reply]
RRMine，雅典娜矿池，全网算力领先1000PB，需要B端C端挖矿方案的滴滴我，欢迎加入社群Wechet：kkmmjjnn00

**Comment by zhaobo443** [comment]
FIL联合封装，用户提供质押币，公司提供设备和运维，收益60％,公司扣出20％服务费和20%设备费，合同期1080天，540天质押币退还，t3天封装。可实地考察。V:zhaobo9308,另接代封装业务，矿池搭建和运维服务，价格美丽，提供教学指导
